recipes/oslo.config/meta.yaml

```diff
@@ -42,7 +42,7 @@ test:
 about:
   home: http://docs.openstack.org/developer/oslo.config/
   license: Apache 2.0
-  # license_file: No MANIFEST.in
+  # license_file: No MANIFEST.in - see https://bugs.launchpad.net/oslo.config/+bug/1624979
```

Comment:
So what's actually missing from upstream?
The LICENSE file is already included in the oslo.config source as far as I can tell:
```console
% wget 'https://pypi.python.org/packages/b3/33/70ec8dfba548cbd562592892a6ae88ee0e3b961805204caa479517133dd0/oslo.config-3.17.0.tar.gz'
% tar zxOf oslo.config-3.17.0.tar.gz oslo.config-3.17.0/LICENSE | head

                                 Apache License
                           Version 2.0, January 2004
                        http://www.apache.org/licenses/

   TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION

   1. Definitions.

      "License" shall mean the terms and conditions for use, reproduction,
```